dotnet/samples/TeamsChefBot/env/.env.dev.user

```diff
@@ -0,0 +1,5 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
```

Comment:
The js sample does not have such .user file. Any preference?

Reply:
I added .user file for user to know the folder structure, just like our TTK samples